resources/views/admin/scholarship/partials/_form_scholarship.blade.php

```diff
@@ -134,6 +134,13 @@
   {!! errorsFor('label_app_essay2', $errors); !!}
 </div>
 
+{{-- Image Uploads --}}
+<div class="form-group">
+  {!! Form::label('image_uploads', 'Allow applicants to upload images?') !!}
```

Comment:
hehe, maybe don't make it a question? `Allow applicants to upload images?` and just a statement, so if checked it is allow it. 🤷‍♂️

Reply:
@weerd I followed how the other checkbox was phrased haha, makes sense to me either way so I don't mind getting rid of it!